helper/testhelpers/ldap/ldaphelper.go

```diff
@@ -0,0 +1,64 @@
+package ldap
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/helper/testhelpers/docker"
+	"github.com/hashicorp/vault/sdk/helper/ldaputil"
+	"github.com/ory/dockertest"
+	"testing"
+)
+
+func PrepareTestContainer(t *testing.T, version string) (cleanup func(), cfg *ldaputil.ConfigEntry) {
```

Comment:
Nice! Thank you for exporting this. I wonder if at some point we'll even want to move it into the `sdk` so it can be reused externally in miscellaneous LDAP plugins.

Reply:
Sure, that makes sense.